client/index.js

```diff
@@ -0,0 +1,2 @@
+//this file exists so the webpack build process will succeed
+Ext._find = require('lodash.find');
```

Comment:
This seems a bit odd. Why would we need to add a lodash method into the Ext object just to get this all working? I don't know a ton about the open tooling, so maybe this is standard but just seems out of place here...

Reply:
@aphex  This file was generated by `ext-gen`, even I validated in other `ext-gen` app, this file is present

Reply:
@aphex I guess this is just one package exposure example.
Take a look
https://docs.sencha.com/extjs/7.5.1/guides/using_systems/using_npm/adding_npm_packages.html#using_systems-_-using_npm/adding_npm_packages_-_2__package_exposure